modules/custom/moj_resources/src/TermApiClass.php

```diff
@@ -59,54 +32,39 @@ public function __construct(
     QueryFactory $entityQuery,
     Serializer $termSerializer
   ) {
-    $this->term_storage = $entityTypeManager->getStorage('taxonomy_term');
-    $this->entity_query = $entityQuery;
+    $this->termStorage = $entityTypeManager->getStorage('taxonomy_term');
     $this->termSerializer = $termSerializer;
```

Comment:
So we use the Drupal serialiser? we serialise our own objects with `createReturnObject`?

Reply:
I actually don't know, it might have been coincidence in that something broke when I removed it 😆